src/blocks/scratch3_control.js

```diff
@@ -73,16 +73,16 @@ Scratch3ControlBlocks.prototype.forever = function (args, util) {
 };
 
 Scratch3ControlBlocks.prototype.wait = function (args, util) {
-    if (!util.stackFrame.timer) {
-        util.stackFrame.timer = new Timer();
-        util.stackFrame.timer.start();
-        util.yield();
-        this.runtime.requestRedraw();
-    } else {
+    if (util.stackFrame.timer) {
         var duration = Math.max(0, 1000 * Cast.toNumber(args.DURATION));
         if (util.stackFrame.timer.timeElapsed() < duration) {
             util.yield();
         }
+    } else {
+        util.stackFrame.timer = new Timer();
+        util.stackFrame.timer.start();
+        util.yield();
+        this.runtime.requestRedraw();
```

Comment:
The rules that emit warnings were ones that I didn't want to disable altogether because I wanted to suggest the more readable style, but also didn't want to enforce completely in cases where it's more readable to break the rule.

E.g., most of the time you want positive conditionals but in this case, I sort of think having the negative conditional first is easier to read since then you have the timer initialization ahead of its usage.  Maybe that use case is enough to just remove the rule and leave it up to reviewers to suggest the other way?